modules/serverless-sinkbinding-odc.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-odc_{context}"]
+= Create a SinkBinding using the Developer perspective
+
+This guide describes the steps required to create, manage, and delete a sample SinkBinding instance by using **Developer** perspective in the {product-title} web console.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have switched to the *Developer* perspective in the web console.
+
+.Procedure
+
+. Create the `event-display` sample Service to use as an event sink.
+.. Switch to the `knative-serving` **Project** in the dropdown menu.
+.. Navigate to **+Add** -> **Container Image**.
++
+image::serverless-odc-sinkbinding-1.png[Add a sample Service event sink as a Container Image]
++
+.. For the **Image** configuration, select **Image name from external registry**, then enter the sample URL:
++
+----
+quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+// TODO: Needs an update to latest sample image for 1.8.0?
++
+The **General** fields, **Application** and **Name**, will be filled automatically once the sample image is provided.
+.. For **Resources**, select **Knative Service**.
+.. Click **Create**.
+. Create a SinkBinding.
+.. Switch to the `knative-serving` **Project** in the dropdown menu.
+.. Navigate to **+Add** -> **Event Source**.
++
+image::serverless-sinkbinding-odc-2.png[Configuring SinkBinding]
++
+.. Select the **SinkBinding** icon.
+.. In the form, enter the following configuration:
+... For **apiVersion** enter `v1`.
+... For **Kind**, enter `SinkBinding`.
```

Comment:
here `apiVersion` and `kind` will be kind of subject as in below deployment can even have Job
See `label selector` to have it working even Deployment needs to be created to proper labels

https://github.com/knative/docs/blob/master/docs/eventing/samples/sinkbinding/sinkbinding.yaml#L19-L25

```
spec:
  subject:
    apiVersion: apps/v1
    kind: Deployment
    selector:
      matchLabels:
        app: wss
```

Reply:
Gonna revisit this and use the CronJob / heartbeats example for now.

We can't really doc every option for `kind` under this PR at least. There are general docs here about CRDs / `kind`: https://docs.openshift.com/container-platform/4.5/operators/crds/crd-extending-api-with-crds.html#crd-custom-resource-definitions_crd-extending-api-with-crds
but they don't mention much about specific types. Maybe there should be a link to https://kubernetes.io/docs/concepts/workloads/controllers/ or similar that explains these concepts without us having to list all the supported types?